File: frontend/src/features/actividades/realizar_revision_previo_firma_banco/models/realizar_revision_previo_firma_banco.ts
export interface RealizarRevisionPrevioFirmaBanco {
  id_realizar_revision_previo_firma_banco: number;
  id_expediente: number;
  enviar_a_reparo: boolean;
  observaciones?: string | null;

  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/realizar_revision_previo_firma_banco.cs
using Common.Domain.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Multibanca.Domain.Models.Multibanca
{
    public class realizar_revision_previo_firma_banco: base_auditoria
    {
        public int id_realizar_revision_previo_firma_banco { get; set; }
        public long id_expediente { get; set; }
        public bool? enviar_a_reparo { get; set; }
        public string? observaciones { get; set; }
    }
}
